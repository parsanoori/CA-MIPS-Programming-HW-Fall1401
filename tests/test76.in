6
523469798
941362867
696647034
475162072
719943903
742348043