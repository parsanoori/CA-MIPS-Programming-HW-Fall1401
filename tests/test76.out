475162072
523469798
696647034
719943903
742348043
941362867